### DISCUSSION TITLE
copy/rename functionality

### DISCUSSION THREAD

**Discussion by node-ray** [enzo1982/freac#281] [General Discussion]
I noticed on the old discussion forum that a copy/rename functionality is in the works, is this still the case? The reason why I need it in particular, is that I have a mix of older mp3 files with various bitrates, that I would rather leave as is- but I have the need to take advantage of the fre:ac functionality to rename folders/files based on the tag data. For example being able to copy/rename albums like <year> - <album> to improve the browsing/listening experience on a car stereo.

**Comment by enzo1982** [comment]
I still have plans to add that functionality at some point, but I'm not currently working on it. So definitely won't be available in the near future, unfortunately.
